src/BeenAnonymizedInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PHPSharkTank\Anonymizer;
+
+interface BeenAnonymizedInterface
+{
+    public function beenAnonymized(): void;
```

Comment:
Maybe `hasBeenAnonymized` would be a better name which sounds more what it should do.